public/js/script_profile.js

```diff
@@ -1,3 +1,71 @@
+//following
```

Comment:
If you're not using jQuery, you could remove (function ($) {})(window.jQuery); from each of these

Reply:
removed the jQuery thing, did I do it correctly?

Reply:
you don't need the (function ($){})() part either,

Reply:
no but i've kept it inside those boxes cause they all share the same variable name for form. I'd suggest we keep it as it is unless it is interfering with anything else on the codebase

Reply:
no but i've kept it inside those boxes cause they all share the same variable name for form. I'd suggest we keep it as it is unless it is interfering with anything else on the codebase